content/agent/basic_agent_usage/amazonlinux.md

```diff
@@ -2,86 +2,161 @@
+| `sudo service datadog-agent stop`               | `sudo service datadog-agent stop`       | Stop Agent running as a service |
+| `sudo service datadog-agent restart`            | `sudo service datadog-agent restart`    | Restart Agent running as a service |
+| `sudo service datadog-agent status`             | `sudo service datadog-agent status`     | Status of Agent service |
+| `sudo service datadog-agent info`               | `sudo datadog-agent status`             | Status page of running Agent |
+| `sudo service datadog-agent flare`              | `sudo datadog-agent flare`              | Send flare |
+| `sudo service datadog-agent`                    | `sudo datadog-agent --help`             | Display command usage |
+| `sudo -u dd-agent -- dd-agent check <check_name>` | `sudo -u dd-agent --datadog-agent check <check_name>` | Run a check |
+{{% /table %}}
 
-To stop the Agent:
+More information about the metrics, events, and service checks for an [integrations](/integrations) can be retrieved with the check command:
 ```shell
-sudo /etc/init.d/datadog-agent stop
+sudo service datadog-agent check [integration]
 ```
 
-To restart the Agent and reload the configuration files:
+Add the check_rate argument to get the most recent values for rates:
 ```shell
-sudo /etc/init.d/datadog-agent restart
+sudo service datadog-agent check [integration] check_rate
 ```
 
-## Status and Information
-To check if the Agent is running: *(since 3.8.0)*
-```shell
-sudo /etc/init.d/datadog-agent status
-```
-To receive information about the Agent’s state:
-```shell
-sudo /etc/init.d/datadog-agent info
-```
+**NB**: If `service` is not available on your system, use:
 
-Tracebacks for errors can be retrieved by setting the `-v` flag: (since 3.8.0)
-```shell
-sudo /etc/init.d/datadog-agent info -v
-```
+* on `upstart`-based systems: `sudo start/stop/restart datadog-agent`
+* on `systemd`-based systems: `sudo systemctl start/stop/restart datadog-agent`
+* on `initctl`-based systems: `sudo initctl start/stop/restart datadog-agent`
 
-More information about the metrics, events, and service checks for an [integrations](/integrations) can be retrieved with the check command:
-```shell
-sudo /etc/init.d/datadog-agent check [integration]
-```
+## Configuration
 
-Add the check_rate argument to get the most recent values for rates:
-```shell
-sudo /etc/init.d/datadog-agent check [integration] check_rate
-```
+The configuration file for the Agent is located at:
 
-## Configuration
-The configuration file for the Agent is located at `/etc/dd-agent/datadog.conf`
+| Agent v5                                  |  Agent v6                          |
+|:-----|:----|
+|`/etc/dd-agent/datadog.conf`| `/etc/datadog-agent/datadog.yaml` |
 
-Configuration files for integrations are located in `/etc/dd-agent/conf.d/
+Configuration files for [integrations](/integrations):
```

Comment:
I'd say "Configuration files and folders for" to stress that in A6 config files will be grouped into `<check_name>.d` folders

Reply:
👌